### ISSUE TITLE
Public Suffix List (PSL) is not ASCII

### ISSUE THREAD

**Issue by alevesely** [whatwg/infra#346]
Section [3.2 "Host miscellaneous"](https://url.spec.whatwg.org/#host-miscellaneous) specifies a host's public suffix.  Bullet 3 says:

    3. Assert: publicSuffix is an ASCII string.

How come?  Since the list is UTF-8 encoded and the algorithm provides no conversions, the resulting *publicSuffix* must be UTF-8.

**Comment by annevk** [comment]
I'm not sure why you filed this against Infra and not URL? In any event, https://publicsuffix.org/list/ under "Formal algorithm" says that all domains are canonicalized to Punycode.

**Comment by mikewest** [comment]
I agree with Anne's read of the PSL page implicitly forcing a punycode conversion as part of the algorithm called from URL. That said, the page also confusingly says "Note: the list uses Unicode, not Punycode forms, and is encoded using UTF-8." which can easily lead to confusion.

@sleevi might know who we could ask to add something more explicit to the algorithm to make the punycodization of inputs (and therefore outputs) more clear.

**Comment by alevesely** [comment]
> why you filed this against Infra and not URL?

I was confused by the [PSL](https://publicsuffix.org/learn/) mentioning "[HTML 5](http://www.w3.org/TR/html5/) (document.domain) ".

>  make the punycodization of inputs (and therefore outputs) more clear.

But then, if a *Public suffix* is formally defined as an ASCII, punycode string (consistent with the [example](https://url.spec.whatwg.org/#example-host-psl)), the *Public Suffix String*, formally is not a list of public suffixes?
